Framework/Algorithms/test/CalMuonDetectorPhasesTest.h

```diff
@@ -124,6 +124,22 @@ class CalMuonDetectorPhasesTest : public CxxTest::TestSuite {
     runExecutionTest(ws);
   }
 
+  /// Test what happens when you supply frequency in Mrad/s rather than MHz
+  void testFrequencyUnits() {
+    auto ws = createWorkspace(4, 100, "Microseconds");
+    auto calc = AlgorithmManager::Instance().create("CalMuonDetectorPhases");
+    calc->initialize();
+    calc->setChild(true);
+    calc->setProperty("InputWorkspace", ws);
+    calc->setPropertyValue("Frequency", "25"); // Mrad/s, not MHz
+    calc->setPropertyValue("DataFitted", "fit");
+    calc->setPropertyValue("DetectorTable", "tab");
+    calc->setProperty("ForwardSpectra", std::vector<int>{1, 2});
+    calc->setProperty("BackwardSpectra", std::vector<int>{3, 4});
+
+    TS_ASSERT_THROWS(calc->execute(), std::runtime_error);
```

Comment:
This assert is failing on clean master builds: http://builds.mantidproject.org/job/master_clean-win7/222/testReport/
@tom-perkins: has there been any related changed that happened more or less in parallel? It would be strange that this passed on the PR builds but then fails on master.